test/e2e/main_test.go

```diff
@@ -31,9 +31,9 @@ func TestMarketplace(t *testing.T) {
 		t.Run("cluster-operator-status-test-group", testgroups.ClusterOperatorTestGroup)
 		testgroups.EnableProxyTests()
 	}
+	t.Run("migration-test-group", testgroups.MigrationTestGroup)
```

Comment:
Is there a reason to change the order?

Reply:
It's described in the commit message. There's an unrelated bug related to the migration code (that I'm in the process of replicating and then I'll create a bugzilla). This is just to avoid the race condition that causes it -- basically we are restarting the operator in the test I added in this pr and the migration test doesn't always handle itself well when the test operator isn't already serving content. We could also in theory put a wait on the status of the opsrc in the test, but I would rather address that holistically in a separate PR that also fixes the migration bug.

Reply:
Sorry I am not able to tie in the commit message to the change here. I must be missing something. Anyhow, a comment here would help too as folks reading the code won't look at the commit message immediately.

Reply:
Sure, I'll quote below:

```
This also adds a test case to the e2e test suite that ensures that the
operator being stopped won't cause this bug to appear in the future.
An issue with an unrelated test case was found with the e2e tests
exposed by the addition of this test (which scales the operator's
deployment down and back up to simulate the restart). That issue
needs to be addressed holistically, so for now this pr simply reorders
the tests to avoid that race condition.
```